src/common/logging.cc

```diff
@@ -79,7 +79,7 @@ void RayLogger_log(RayLogger *logger,
 
     int status =
         redisAsyncCommand(context, NULL, NULL, formatted_message,
-                          (char *) db->client.id, sizeof(db->client.id));
+                          (char *) db->client.data(), sizeof(db->client));
```

Comment:
we should be using C++ style casts everywhere (at least for new code)

Reply:
let's do this incrementally